src/azul/service/elasticsearch_service.py

```diff
@@ -99,6 +100,15 @@ def _translate_filters(self, filters: Filters, field_mapping: JSON) -> MutableFi
             assert isinstance(value, dict)
             assert isinstance(one(value.values()), list)
             field_type = self.field_type(tuple(key.split('.')))
+            if key == 'contents.donors.organism_age':
+                metadata_plugin = self.metadata_plugin
+                assert isinstance(metadata_plugin, Plugin)
```

Comment:
Can you make it so this reads

```suggestion
                assert isinstance(metadata_plugin, hca.Plugin)
```